modules/task.js

```diff
@@ -0,0 +1,53 @@
+class Task {
+  constructor(description, index) {
+    this.description = description;
+    this.completed = false;
+    this.index = index;
+  }
+
+  Add = () => {
+    let tasks = [];
+    const storage = JSON.parse(localStorage.getItem('to_do_list'));
+    if (storage !== null) {
+      tasks = storage;
+    }
```

Comment:
I saw in every file, you are getting the local storage `to_do_list` value and then checking for null. Because you are repeating the code again and again. I recommend you create a function and return it.